pkgs/tools/misc/coreboot-configurator/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub, inkscape, meson, ninja, pkg-config, libyamlcpp, nvramtool, qt5 }:
+
+stdenv.mkDerivation {
+  pname = "coreboot-configurator";
+  version = "2022-08-22";
+
+  src = fetchFromGitHub {
+    owner = "StarLabsLtd";
+    repo = "coreboot-configurator";
+    rev = "37c93e7e101a20f85be309904177b9404875cfd8";
+    sha256 = "2pk+uJk1EnVNO2vO1zF9Q6TLpij69iRdr5DFiNcZlM0=";
+  };
+
+  nativeBuildInputs = [ inkscape meson ninja pkg-config qt5.wrapQtAppsHook ];
+  buildInputs = [ libyamlcpp qt5.qtbase qt5.qtsvg ];
+
+  postPatch = ''
+    substituteInPlace src/application/*.cpp \
+      --replace '/usr/bin/pkexec' '/run/wrappers/bin/sudo' \
```

Comment:
This largely assumes that the user uses `sudo`, and not `doas` or similar. Is there another way we can do this without hardcoding this stuff?

Reply:
Also this only works on NixOS and not when using nixpkgs on other distros.

Reply:
The application uses a hardcoded path so we need to choose one to substitute in.

I'm not sure how we can make it work simultaneously on NixOS and other distros, as they will require a different path.

Reply:
Do we really must use a full path and cannot rely to resolve this via PATH?

Reply:
Actually it does work using PATH. Good suggestion.